src/site/static/eligibility.liquid

```diff
@@ -2002,46 +2002,55 @@ pageClass: "page-public-assistance"
-          document.getElementById("existing-calworks-me").checked,
-          document.getElementById("existing-calworks-household").checked,
-          document.getElementById("existing-calfresh-me").checked,
-          document.getElementById("existing-calfresh-household").checked,
-          document.getElementById("existing-medical-me").checked,
-          document.getElementById("existing-medical-household").checked,
-          document.getElementById("existing-cfap-me").checked,
-          document.getElementById("existing-cfap-household").checked,
-          document.getElementById("existing-nslp-me").checked,
-          document.getElementById("existing-nslp-household").checked,
-          ssiResult().eligible,
-          liheapResult().eligible,
-          wicResult().eligible,
-          calworksResult().eligible,
-          calfreshResult().eligible));
+    const paysUtilities = document.getElementById("pay-utilities-yes").checked;
+    
+    const incomeLimit = careIncomeLimit().getLimit(householdSize());
+    const underIncomeLimit = le(grossIncome(), incomeLimit);
+
+    const isCategoricallyEligible = or(
+      // TODO: add remaining assistance programs that help qualify.
+      document.getElementById("existing-ssi-me").checked,
+      document.getElementById("existing-ssi-household").checked,
+      document.getElementById("existing-liheap-me").checked,
+      document.getElementById("existing-liheap-household").checked,
+      document.getElementById("existing-wic-me").checked,
+      document.getElementById("existing-wic-household").checked,
+      document.getElementById("existing-calworks-me").checked,
+      document.getElementById("existing-calworks-household").checked,
+      document.getElementById("existing-calfresh-me").checked,
+      document.getElementById("existing-calfresh-household").checked,
+      document.getElementById("existing-medical-me").checked,
+      document.getElementById("existing-medical-household").checked,
+      document.getElementById("existing-cfap-me").checked,
+      document.getElementById("existing-cfap-household").checked,
+      document.getElementById("existing-nslp-me").checked,
+      document.getElementById("existing-nslp-household").checked,
+      ssiResult().eligible,
+      liheapResult().eligible,
+      wicResult().eligible,
+      calworksResult().eligible,
+      calfreshResult().eligible);
 
     const program = new Program();
-    // TODO: Replace this single example condition with a set of simplified
-    // conditions describing the separate eligibility requirements.
-    program.addCondition(new EligCondition('Example', eligible));
+    program.addCondition(
+      new EligCondition('Is housed', isHoused));
+    program.addCondition(
+      new EligCondition('Pays utilities', paysUtilities));
+    program.addConditionsOneOf([
+      new EligCondition(`Adjusted income is below ${usdLimit(incomeLimit)} per month`,
```

Comment:
The term `Gross income` here is more accurate than `Adjusted income` because there are no adjustments (deductions, disregards, etc) made to the gross income value before comparing it to the limit.